be/src/vec/columns/predicate_column.h

```diff
@@ -447,13 +459,9 @@ class PredicateColumnType final : public COWHelper<IColumn, PredicateColumnType<
         // DateV1 and DateTimeV1 is special, its storage format is different from compute format
         // should convert here.
         if constexpr (Type == TYPE_DATE || Type == TYPE_DATETIME) {
-            if constexpr (std::is_same_v<T, uint32_t>) {
-                insert_date_to_res_column(sel, sel_size, column);
-            } else if constexpr (std::is_same_v<T, uint64_t>) {
-                insert_datetime_to_res_column(sel, sel_size, column);
-            } else {
-                LOG(FATAL) << "not reachable";
-            }
+            insert_datetime_to_res_column(sel, sel_size, column);
```

Comment:
If datev1 using the same type in predicate and compute layer, why do we need use a specific method here?